zeppelin-server/pom.xml

```diff
@@ -165,6 +165,17 @@
     <dependency>
       <groupId>org.apache.shiro</groupId>
       <artifactId>shiro-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-beanutils</groupId>
+          <artifactId>commons-beanutils</artifactId>
```

Comment:
This version of Shiro downloads BeanUtils-1.8.3, which is marked with CVE-2014-0114. Hence, bumping up version of this as well.